package.json

```diff
@@ -51,7 +51,7 @@
     "rollup": "^0.41.6",
     "standard-version": "^4.0.0",
     "tslint": "^5.1.0",
-    "typescript": "^2.3.2",
+    "typescript": "^2.4.0",
```

Comment:
2.4 is released. It's published as the latest on npm, so it's in the wild. 2.4.1 actually just dropped.

Figured we might as well bump it up since it works fine and then provides a meaningful build verification.